.github/workflows/AutoCreateMasterBranch.yml

```
@@ -0,0 +1,107 @@
+name: AutoCreateMasterBranch
+
+on:
+  create:
+    branches:
```

actionlint (1.7.8)

"branches" filter is not available for create event. it is only for merge_group, push, pull_request, pull_request_target, workflow_run events

(events)